spec/include_directive/parser_spec.rb

```diff
@@ -1,3 +1,4 @@
+require_relative '../../spec/spec_helper'
```

Comment:
You're right. Not sure what I did when I tested it first, but the simple case does indeed work.